src/Form/FormAnalyzer.js

```diff
@@ -46,7 +46,7 @@ class FormAnalyzer {
     }) {
         const negativeRegex = new RegExp(/sign(ing)?.?in(?!g)|log.?in/i)
         const positiveRegex = new RegExp(
-            /sign(ing)?.?up|join|regist(er|ration)|newsletter|subscri(be|ption)|contact|create|start|settings|preferences|profile|update|checkout|guest|purchase|buy|order|schedule|estimate/i
+            /sign(ing)?.?up|join|regist(er|ration)|newsletter|subscri(be|ption)|contact|create|start|settings|preferences|profile|update|checkout|guest|purchase|buy|order|schedule|estimate|request/i
```

Comment:
I can't claim to understand what these mean but I'm sure they're based on testing. IMO we should move this to https://github.com/duckduckgo/privacy-configuration/ as a follow up.

Reply:
Other than safari yes.

Reply:
https://app.asana.com/0/1198964220583541/1201323323767696/f